docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+          </listitem>
+
+          <listitem>
+            <para>Change directory to the modules/storage_account
+            directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Run terraform init in that directory:</para>
+
+            <programlisting>terraform init</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Confirm the module has been successfully initialized.</para>
+          </listitem>
+
+          <listitem>
+            <para>Change directory to the root,
+            terraform-azurerm-hpcc directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Run terraform init in that directory:</para>
+
+            <programlisting>terraform init</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Confirm the module has been successfully initialized.</para>
+          </listitem>
+        </orderedlist>
+      </sect3>
+
+      <sect3 id="ApplyingTheTerraformModules">
+        <title>Applying the Terraform Modules</title>
+
+        <para>This step generates a Terraform Plan to confirm your
+        configuration choices. A Terraform plan is where after having
+        initialized and readied all the modules displays for your review
+        exactly what it is going to do. You can review and either approve to
+        implement the plan or abort the plan and review your configuration
+        modules for further changes.</para>
+
+        <para>When you issue the Terraform apply command it will validate the
+        Terraform code and generate the plan, which you will then accept or
+        reject to proceed. As with the previous steps, the order the modules
+        are applied is important. You must apply the virtual network first,
+        then the storage, and finally the root.</para>
+
+        <para>To Generate a Plan and Apply the Modules: </para>
+
+        <orderedlist numeration="arabic">
+          <listitem>
+            <para>Change directory to the modules/virtual_network
+            directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Run Terraform apply, specifying to use the admin.tfvars file
+            you configured previously. </para>
+
+            <programlisting>terraform apply –var-file=admin.tfvars </programlisting>
+          </listitem>
+
+          <listitem>
+            <para>The Terraform plan displays, review the plan and if it
+            aligns with what you expected, approve the plan and enter
+            yes.</para>
+
+            <para><emphasis role="bold">Note</emphasis>: If something does not
+            look correct, do not enter yes. Anything other than yes will abort
+            the application. Then go back and re-examine the admin.tfvars file
+            from the previous steps. Make any necessary changes.</para>
+          </listitem>
+
+          <listitem>
+            <para>Wait for confirmation. Terraform will now go about and
```

Comment:
Terraform initializes all the declared resources until they are all in a ready state. This can take a little time, as it is initializing several resources.